pkg/minishift/registration/utils.go

```diff
@@ -26,12 +26,18 @@ import (
 )
 
 type RegistrationParameters struct {
-	Username string
-	Password string
+	Username           string
+	Password           string
+	SkipRegistration   bool
+	SkipUnRegistration bool
 }
 
 // Register host VM
 func RegisterHostVM(host *host.Host, param *RegistrationParameters) error {
+	if param.SkipRegistration {
+		log.Debug("User choosed to skip registration")
```

Comment:
@praveenkumar partly agree. I think I should have been more explicit. The part which we think implicit is user knows that without registration things will not work while deploying application OpenShift cluster (in Minishift). But I believe not all users know this, hence we need to communicate this to user.